src/java.base/share/classes/java/io/FileInputStream.java

```diff
@@ -115,10 +114,8 @@ public FileInputStream(String name) throws FileNotFoundException {
     }
 
     /**
-     * Creates a {@code FileInputStream} by
-     * opening a connection to an actual file,
-     * the file named by the {@code File}
-     * object {@code file} in the file system.
+     * Creates a {@code FileInputStream} to read from an existing file
+     * named by the {@code File} object {@code file}.
```

Comment:
Hello Brian, should be perhaps reword this as "... to read from an existing file represented by the {@code file}." This would almost match what we state for the constructor which takes the `FileDescriptor` instance.

Reply:
Updated for the next commit.

Reply:
Did you push that commit? I don't see the update, the PR only has 1 commit.

Reply:
I did just now.